api/src/services/lottery.service.ts

```diff
@@ -426,22 +447,46 @@ export class LotteryService {
    * @param csvHeaders the headers and renderers of the export
    * @param queryParams the incoming param args
    * @param forLottery whether we are getting the lottery results or not
+   * @param preference if present, then builds the preference specific spreadsheet page
    * @returns void but writes the output to a file
    */
-  async lotteryExportHelper(
+  async generateSpreadsheetData(
     workbook: Excel.Workbook,
     applications: Application[],
     csvHeaders: CsvHeader[],
     queryParams: ApplicationCsvQueryParams,
     forLottery = false,
+    preference?: IdDTO,
   ): Promise<void> {
-    // create raw rank spreadsheet
-    const rawRankSpreadsheet = workbook.addWorksheet('Raw');
-    rawRankSpreadsheet.columns = this.buildExportColumns(csvHeaders);
+    // create a spreadsheet. If the preference is passed in use that as a title otherwise 'raw'
+    const spreadsheet = workbook.addWorksheet(
+      preference ? preference.name : 'Raw',
```

Comment:
The sample export has the title as `Raw Lottery Rank`

Reply:
fixed!

Reply:
It seems like you changed the wrong header name :/
<img width="495" alt="Screenshot 2024-08-12 at 11 43 23 AM" src="https://github.com/user-attachments/assets/f2f36ae4-6771-4b24-a162-81971a5412f7">